### PR TITLE
Solve all travis 

### PR DESCRIPTION
**Pull request by rlodev67**
## Description

This PR is cleaning all bad things to let travis to his job;

## Motivation and Context

Since I asked to @SenorKarlos  to add travis to avoid bad things to be commited, we need to do this job.
## How Has This Been Tested?

Not tested, it have to be tested, especially : 
- dynimg
- no dynimg
- scout request
- notify with popup
- with pgpool / without pgpool

since hashing is down at the moment, it can be tested by manually adding pokemon

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by darkelement1987** [comment]
[03-23 23:53:37] [      MainThread] [        models] [   ERROR] Your database version (30) appears to be newer than the code supports (29).
 [03-23 23:53:37] [      MainThread] [        models] [   ERROR] Please upgrade your code base or drop all tables in your database.

**Comment by darkelement1987** [comment]
- Notifications popups working

**Comment by darkelement1987** [comment]
- Dynimg working:

image

File: pogom/app.py [hunk 1]
```diff
@@ -352,16 +367,15 @@ def fullmap(self, statusname=None):
...
-                               )
+        return render_template(
+            'map.html',
+            lat=map_lat,
+            lng=map_lng,
+            showAllZoomLevel=args.show_all_zoom_level,
+            generateImages=str(args.generate_images).lower(),
+            gmaps_key=args.gmaps_key,
+            lang=args.locale,
+            show=visibility_flags)
```
Comment (rlodev67) [inline pogom/app.py]: Missing rarity_filename

File: pogom/app.py [hunk 2]
```diff
@@ -449,31 +463,21 @@ def raw_data(self):
...
-            # Exclude ids of Pokemon that are hidden.
-            eids = []
-            request_eids = request.args.get('eids')
-            if request_eids:
-                eids = {int(i) for i in request_eids.split(',')}
-
             if request.args.get('ids'):
-                request_ids = request.args.get('ids').split(',')
-                ids = [int(x) for x in request_ids if int(x) not in eids]
+                ids = [int(x) for x in request.args.get('ids').split(',')]
```
Comment (rlodev67) [inline pogom/app.py]: Missing #85

File: pogom/app.py [hunk 3]
```diff
@@ -576,14 +584,8 @@ def raw_data(self):
...
-                max_status_age = args.status_page_filter
-                if max_status_age > 0:
-                    d['main_workers'] = MainWorker.get_recent(max_status_age)
-                    d['workers'] = WorkerStatus.get_recent(max_status_age)
-                else:
-                    d['main_workers'] = MainWorker.get_all()
-                    d['workers'] = WorkerStatus.get_all()
-
+                d['main_workers'] = MainWorker.get_all()
+                d['workers'] = WorkerStatus.get_all()
```
Comment (rlodev67) [inline pogom/app.py]: Missing #73

**Pull request merged**